tests/acceptance/tests/basic_dsc_resources/waitforall/negative/waitforall_blacklist.rb

```diff
@@ -18,7 +18,7 @@
 dsc_manifest = ERB.new(File.read(dsc_manifest_template_path), 0, '>').result(binding)
 
 # Verify
-error_msg = /Error:.*Invalid resource type dsc_waitforall/
+error_msg = /Error:.*resource.*dsc_waitforall/
```

Comment:
Or to put the argument another way, for the same reason "Invalid" in the statement "Invalid resource type" no longer works. Because it is another word that could change, introduce punctuation, or contain line feeds between. Granted, it's very unlikely to change. 

Of course we'd hoped that "Invalid" didn't change and it did, although I think it makes more sense in context that it is now "Unknown" instead. I would just rather not see tests fail when someone decides to change wording.